src/ioc_scan/stix_extract.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env python3
+
+"""
```

Comment:
This should not be included in a file that is part of a package. The `ioc_scanner.py` file is an unpleasant hacky thing that does not conform to our general Python package guidelines and I see no rationale for this module to be configured similarly.

```suggestion
"""
```

Reply:
Hi @mcdonnnj , Is the suggested change being made? I still notice in the file stix_extract.py the first line reads as #!/usr/bin/env python3.